core/block.ex

```diff
@@ -61,7 +61,7 @@ defmodule UltraDark.Blockchain.Block do
     # to the point where mining performance gets HALVED
     IO.write "Block Index: #{index} -- Hash: #{hash} -- Nonce: #{nonce}\r"
 
-    block = %{ block | hash: Utilities.sha_base16([Integer.to_string(index), previous_hash,  timestamp, Integer.to_string(nonce)]) }
+    block = %{ block | hash: Utilities.sha3_hashing([Integer.to_string(index), previous_hash,  timestamp, Integer.to_string(nonce)]) }
```

Comment:
Let's pull in the latest from master, @Zac-Garby added merkle_root to the contents that are being hashed, and merging this in would overwrite it